foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -109,6 +109,22 @@
             "includes": ["src/**/*.java", "src/**/*.properties"]
           }
         ]
+      },
+      {
+        "name": "upgrade-project-specific-image-naming-convention-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_7_0.ProjectSpecificImageNameMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "*-deploy/src/main/resources/apps/*/values.yaml",
+              "*-deploy/src/main/resources/apps/spark-worker-image/spark-worker-image.yaml",
+              "*-deploy/src/main/resources/apps/aissemble-machine-learning-training-image/aissemble-machine-learning-training-image.yaml",
+              "*-deploy/src/main/resources/apps/airflow/values-dev.yaml",
+              "*-pipelines/*/src/main/resources/apps/*-base-values.yaml",
```

Comment:
C: Also need this for capturing pyspark base-values:
```
"*-pipelines/*/src/*/resources/apps/*-base-values.yaml",
```

Reply:
Ah good catch, didn't realize python had the extra directory. Will add

Reply:
Added pyspark base-values file ✅

Reply:
Q: Could you just opening it up to `"*-deploy/src/main/resources/apps/*/values*.yaml"` instead? Thinkin we would want to catch this in any values files if it's there.

Reply:
Yea we can do that. From what I saw the image name was defined in the `values.yaml` file but I'll open it up to catch any edge cases if they're out there